docs/features/shared-volumes-with-NFS.md

```diff
@@ -0,0 +1,138 @@
+"LogLevel": "debug",
+_“RemoteDirs”: {_
+     _“<dir-A-label>” :   {_
+          _“addr”: <IP>,_
+          _“args: <mount args>,_
+          _“path”: <remote path to mount>,_
+          _“type”: <FS type – nfs3, nfs4, efs, cifs, etc>,_
+          _“vol-path”: <optional path name under path above to locate volumes>_
+	  _"src": <label of another entry in this list to use to locate volumes>_
+	  _"default": <0/1>_
+     _},_
+     _“<dir-B-label>” :   {_
+          _“addr”: <IP>,_
+          _“args: <mount args>,_
+          _“path”: <remote path to mount>,_
+          _“type”: <FS type – nfs3, nfs4, efs, cifs, etc>,_
+          _“vol-path”: <optional path name under path above to locate volumes>_
+	  _"src": <label of another entry in this list to use to locate volumes>_
+	  _"default": <0/1>_
+     _},_
+     _},_
+     _“<dir-C-label>” :   {_
+          _“addr”: "",_
+          _“args: "",_
+          _“path”: "",_
+          _“type”: "",_
+          _“vol-path”: <optional path name under path above to locate volumes>_
+	  _"src": "dir-A-label"_
+	  _"default": <0/1>_
+     _},_
+     _},_
+  _}_
+}
+
+The configuration items are described below:
+
+1. “RemoteDirs”:  Map of remote dirs.
+1. “<dir-A-label>” – must be unique and user defined and identifies a specific remote dir or another entry within the list
+1. “addr” – IP or fully qualified host name of server
+1. “args – args to mount remote dir.
+1. “path” – path on server to mount
+1. “type” – FS type of remote dir.
+1. “vol-path” – a path name that is relative to the mounted path abnove, volumes will be create relative to this path if non-null, if not specified then volumes are located under path above. The _vol-path_ entry allows the user  user to separate volumes for different docker service stacks within a single remote dir.
+1. "src" - indicates that a remote-dir (identified by the assigned label)in the list be used to locate volumes. Combined with a _vol-path_ allows the user to direct placement of volumes on separate folders within a single remote dir. Especially useful when the user wants to separate volumes for different service stacks into separate folders.
+1. "default" - If a label isn't specified at creation time of a volume for _file_ type volumes then this entry will be used to select the location for the volume.
+
+# Volumes
+
+1. Provisioned as a folder on the named network FS
+1. /mnt/vmdk/<remote—dir-label>/<vol-path>/<vol-name>/
+1. Volume _create_ mkdir’s the volume name folder if not already present
+
+## Volume meta-data
+Plugin maintains no in-memory state for either volumes or remote dirs. (except the configuration). Volume met-data listed below are generated when requested via a Get() from Docker,
+1. _Location_ – remote-dir, path within remote-dir
+1. _Type and version_ - volume type (NFS, EFS)
+1. _Capacity used in FS blocks_
+1. _Current usage count-
+1. _Create date/time_
+1. _Modified time_
+1. _Access time_
+1. _Access permissions_
+1. _Use count_
```

Comment:
This is just the local refcount for the volumes mounted to containers from the refcounter.